### ISSUE TITLE
Bw2Color with Pix2Pix not working

### ISSUE THREAD

**Issue by shak1** [junyanz/pytorch-CycleGAN-and-pix2pix#1200]
Hi
I'm implementing the Pix2Pix architecture with keras following this [tutorial](https://www.tensorflow.org/tutorials/generative/pix2pix#top_of_page) on tensorflow site for the colorization task. I created a dataset from ImageNet by randomly choosing 25 classes(mainly animals) and for each class i downloaded 1000 images so to have a dataset with 25k images.
i trained the model for 150 epochs with 256x256 images, learning rate of 0.00002 , batch size of 4 and then i tried to colorize some images but  the results weren't so good. For example
![test](https://user-images.githubusercontent.com/10812812/100370404-812bcb00-3006-11eb-8f68-b2e0b5f9a4ce.png)

As you can see it looks like the generator randomly colored the image. What could be the problem ?

1. The dataset is too small ?
2. Should i train the model more ?
3. or change the hyper parameters(learning rate, batch size, loss function. ecc) ?

Thanks
